apps/mark-scan/backend/src/custom-paper-handler/state_machine.ts

```diff
@@ -1286,29 +1272,11 @@ export async function getPaperHandlerStateMachine({
       });
     },
 
-    async printBallot(pdfData: Buffer): Promise<void> {
-      if (
-        isFeatureFlagEnabled(
-          BooleanEnvironmentVariableName.USE_MOCK_PAPER_HANDLER
-        )
-      ) {
-        const pdfPages = iter(pdfToImages(pdfData));
-        const pdfPage = await pdfPages.first();
-        assert(pdfPage);
-
-        const jpgPath = tmpNameSync({ postfix: '.jpg' });
-        await writeImageData(jpgPath, pdfPage.page);
-
-        machineService.send({
-          type: 'SET_INTERPRETATION_FIXTURE',
```

Comment:
noting that this state can probably be replaced with usage of [`setMockPaperContents`](https://github.com/votingworks/vxsuite/pull/4979/files#diff-a61779136fdb9187d8346134eb8b69030bb10931fc271581e01e8a03afe47c2cR318) in the future (doesn't need to be in this PR or by you in particular)